practice/src/ts/templates/dropDownTemplate.ts

```diff
@@ -0,0 +1,27 @@
+function genBtn(): string {
+  return `
+    <div class="btn-dropdown">
+      <div class="dot"></div>
+      <div class="dot"></div>
+      <div class="dot"></div>
+    </div>
```

Comment:
IMO, If a template is simple, doesn't bind values, and isn't reused, you don't need to separate it.